bitnami/etcd/templates/scripts-configmap.yaml

```diff
@@ -130,13 +131,15 @@ data:
               --initial-cluster $ETCD_INITIAL_CLUSTER \
               --initial-cluster-token $ETCD_INITIAL_CLUSTER_TOKEN \
               --initial-advertise-peer-urls $ETCD_INITIAL_ADVERTISE_PEER_URLS 1>&3 2>&4
-            store_member_id &
+            store_member_id & 1>&3 2>&4
+
```

Comment:
Please remove extra lines. This apply to the rest of changes too.

Reply:
Good idea, done!